### ISSUE TITLE
uninitialized constant Net::DNS::Resolver::TimeoutError

### ISSUE THREAD

**Issue by ujinius** [bluemonk/net-dns#84]
when timeout is happening I got:

`NameError (uninitialized constant Net::DNS::Resolver::TimeoutError)`

It can be reproduced simply:

```
irb(main):023:0> Net::DNS::Resolver.new({ retry_number: 3, udp_timeout: 0.0001 }).search("8.8.4.4".reverse, Net::DNS::PTR)
Traceback (most recent call last):
        1: from (irb):23
NameError (uninitialized constant Net::DNS::Resolver::TimeoutError)
irb(main):024:0> 
```

**Comment by ujinius** [comment]
it seems like this issue will be solved when this PR will be applied - https://github.com/bluemonk/net-dns/pull/83

**Comment by akahn** [comment]
Looks like the above has been merged. @bluemonk, would it be possible to cut a new gem release with this fix? Many thanks!

**Comment by akahn** [comment]
@ujinius what version of Ruby are you on?

**Comment by ujinius** [comment]
> @ujinius what version of Ruby are you on?

@akahn 3.1.4 now

**Comment by weppos** [comment]
Fixed by https://github.com/bluemonk/net-dns/pull/83

**Comment by weppos** [comment]
> Looks like the above has been merged. @bluemonk, would it be possible to cut a new gem release with this fix? Many thanks!

Published 0.20.0 https://rubygems.org/gems/net-dns/versions/0.20.0
